dice_ml/diverse_counterfactuals.py

```diff
@@ -29,16 +29,17 @@ def convert_to_dataframe(self):
         org_instance = org_instance[self.data_interface.feature_names + [self.data_interface.outcome_name]]
         self.org_instance = self.data_interface.de_normalize_data(org_instance)
 
+        precisions = self.data_interface.get_decimal_precisions()
```

Comment:
I understand you are setting float precision. Why is it needed? Will be good to add code comment to justify.

Reply:
Yes added. It is to display the CFs and test instances with the same precision as the original data.